### ISSUE TITLE
Homepage revamp

### ISSUE THREAD

**Issue by yangshun** [facebook/docusaurus#6386]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#reporting-new-issues).

### Description

The homepage hasn't changed since I built it 2 years ago and I think it's time for a revamp. I'm not a designer so I'm taking inspiration from some of my favorite websites (https://turborepo.org, https://vercel.com, https://supabase.com) and imitating their structure. This issue serves as a discussion for the content structure. With a content structure I could do up the design for the new homepage.

## Hero/Tagline

Doesn't have to change much, but we should discuss with @JoelMarcey and see if the current tagline is still relevant

## Features 

With v2, with have so many more features, maybe we should relook at this section and also change the design a little. 

## Projects/Orgs using it

A list of logos of projects/orgs using it. 

- Orgs: Meta, Microsoft, Amazon, Redis, Spotify, Supabase
- Projects: React Native, Jest, Relay, Redux

Suggestions welcome!

## Testimonials

The current testimonials are kinda outdated and are for Docusaurus v1. Displaying Tweets for v2 might be more relevant and appeal better as they are shorter and easier to read. Inspiration: https://supabase.com and https://turborepo.org/

We could take the first 10 and have a separate page (/testimonials) for the rest, and continuously update when we see new tweets.

- https://twitter.com/kentcdodds/status/1323806816019468288
- https://twitter.com/swyx/status/1454261201207054339
- https://twitter.com/paularmstrong/status/1387059593373700100
- https://twitter.com/sebastienlorber/status/1321784071815680000
- https://twitter.com/rachelnabors/status/1321787416089366529
- https://twitter.com/mweststrate/status/1181276252293853186
- https://twitter.com/acemarke/status/1420554029852241920
- https://twitter.com/yangshunz/status/1284536949718478848
- https://twitter.com/biantris_/status/1480259279487741953
- https://twitter.com/verdaccio_npm/status/1420187249145118722
- https://twitter.com/_aantipov/status/1384479759531495424
- https://twitter.com/iansu/status/1184149586048245760
- https://twitter.com/MThakkar_/status/972900687527866368
- https://twitter.com/pierregillesl/status/1372839188698001408
- https://twitter.com/johnny_reilly/status/1469238609266028545

cc @slorber @JoelMarcey @Josh-Cena 

### Self-service

- [X] I'd be willing to address this documentation request myself.

**Comment by slorber** [comment]
Agree, we should refresh it!

> ### Description
> The homepage hasn't changed since I built it 2 years ago and I think it's time for a revamp. I'm not a designer so I'm taking inspiration from some of my favorite websites ([turborepo.org](https://turborepo.org), [vercel.com](https://vercel.com), [supabase.com](https://supabase.com)) and imitating their structure. This issue serves as a discussion for the content structure. With a content structure I could do up the design for the new homepage.


Agree, let's focus on the content here.

Regarding the copy, I tried to make it clear and concise what Docusaurus is in the intro (https://docusaurus.io/docs), but left the landing page untouched for now

Regarding the design, maybe we could create a separate issue?

Still some UI thoughts:
- Gradients are trendy
- I feel like we don't dogfood enough site customizations, ensuring robustness, flexibility and detecting unwanted side-effects earlier. I'd like the Docusaurus website to be much more customized and beautiful in the future.


## Hero/Tagline

> Doesn't have to change much, but we should discuss with @JoelMarcey and see if the current tagline is still relevant

Regarding the copy, we probably want to focus more on docs, blog and pages more than on "any website"

We can probably remove the v1=>v2 migration banner soon or find something else to say there
 
## Features

> With v2, with have so many more features, maybe we should relook at this section and also change the design a little.

We have more features but those 5 ones are still relevant to me, not sure it's useful to bloat the homepage with a very long list. What do you have in mind in terms of other features to highlight?


# Projects/Orgs using it

> * Orgs: Meta, Microsoft, Amazon, Redis, Spotify, Supabase

Many large orgs that are not always on our site showcase: LinkedIn, SAP, Lacework, Palo Alto, JPMorgan...

> * Projects: React Native, Jest, Relay, Redux

Even if we are better suited for React/MDX/JS users, we have a larger user base.

I don't like much the idea of only showcasing React-related projects on our homepage, but would rather add a more diverse list. The idea for me is that you should see a familiar project/company in this list even if you are not familiar with React. We have a lot of projects that a dev might still know or heard of: Solana, Hashnode, Supabase...


> ## Testimonials
> The current testimonials are kinda outdated and are for Docusaurus v1. Displaying Tweets for v2 might be more relevant and appeal better as they are shorter and easier to read. Inspiration: [supabase.com](https://supabase.com) and [turborepo.org](https://turborepo.org/)

Agree, we should update those testimonials. I also think it's nice to be able to link to a tweet so that visitors can check that it's not a fake testimonial

> We could take the first 10 and have a separate page (/testimonials) for the rest, and continuously update when we see new tweets.

We can see if it's possible to automate this, but I think we should curate the top testimonials for the homepage and the top of the testimonials page

**Comment by Josh-Cena** [comment]
> remove the v1=>v2 migration banner soon or find something else to say there

Not until we release v3. I anticipate a lot of projects to migrate to v2 only after we have officially released (e.g. KaTeX, babel, prettier)

**Comment by slorber** [comment]
Something I've read a few times: "oh I didn't know Docusaurus was from Facebook". 

Maybe it's worth making this more visible on the homepage (FB/Meta only appears in the footer atm), as developers assessing doc tools are more likely to adopt Docusaurus if they know it's backed by a large company

**Comment by slorber** [comment]
To consider: how/where to embed the Meta ELI5 video (cf https://github.com/facebook/docusaurus/pull/6659)

**Comment by PatelN123** [comment]
An idea could be having a video like so on the main page, just customised to docusaurus. 

https://user-images.githubusercontent.com/79650289/154807216-a35ec64d-6fc3-459b-a949-0b3b3d3c4eef.mp4

Maybe it could start with the config file, and configuring the logo and site name then the page switches to a md file...

**Comment by Josh-Cena** [comment]
Note as well that the lighthouse performance score on our landing page is constantly degrading... (currently staying at ~50) We should find a way to make sure it doesn't regress.
